x-pack/legacy/plugins/apm/public/components/shared/TimestampTooltip/index.test.tsx

```diff
@@ -7,17 +7,54 @@
 import { shallow } from 'enzyme';
 import React from 'react';
 import moment from 'moment-timezone';
-import { TimestampTooltip } from './index';
+import { TimestampTooltip, asAbsoluteTime } from './index';
 import { mockNow } from '../../../utils/testHelpers';
 
+describe('asAbsoluteTime', () => {
```

Comment:
Might be good to do `afterAll(() => moment.tz.setDefault(''))` to ensure we don't leave a specific state.